### PR TITLE
docs: add link to a proper aws guide for configuring credentials

### PR DESCRIPTION
**Pull request by genbit**
Add a link on top of the credentials docs to a proper aws guide

Signed-off-by: Sergey Generalov <sergey@genbit.ru>

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Comment by dannyrandall** [comment]
could you add https://github.com/aws/copilot-cli/pull/4089#discussion_r995903495 and https://github.com/aws/copilot-cli/pull/4089#discussion_r995903680? i forgot to add the label to stop them from merging in that PR 🥺

**Comment by genbit** [comment]
@dannyrandall done

File: site/content/docs/credentials.en.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-This section explains our recommendations around credentials to provide the best experience with the AWS Copilot CLI.
+AWS Copilot uses AWS credentials to access AWS API, store and look up an [application's metadata](concepts/applications.en.md), and deploy and operate application's workloads.
+
+You can learn more on how to configure AWS credentials at [Quick configuration with `aws configure`](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config).
```
Comment (huanjani) [inline site/content/docs/credentials.en.md]: This is super helpful for first-time users. I remember being confused by this when I first started! Thank you.

File: site/content/docs/credentials.en.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-This section explains our recommendations around credentials to provide the best experience with the AWS Copilot CLI.
+AWS Copilot uses AWS credentials to access AWS API, store and look up an [application's metadata](concepts/applications.en.md), and deploy and operate application's workloads.
```
Comment (huanjani) [inline site/content/docs/credentials.en.md]:
```suggestion
AWS Copilot uses AWS credentials to access the AWS API, store and look up an [application's metadata](concepts/applications.en.md), and deploy and operate an application's workloads.
```

File: site/content/docs/credentials.en.md (hunk 1 above)
Comment (dannyrandall) [inline site/content/docs/credentials.en.md]:
```suggestion
You can learn more on how to configure AWS credentials in the [AWS CLI's documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html).
```

**Review by Lou1415926 — APPROVED** [review]

**Review by huanjani — APPROVED** [review]

**Pull request merged**